docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    COLLECTIONID = 'C1202077968-NASA_MAAP' # specify the collection id for the ATLAS dataset
</pre><p>I would use the collection variable you defined above here to get this id programmatically - i.e. <code>collection_concept_id = colllection['concept-id']</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>